desk/app/groups.hoon

```diff
@@ -2976,12 +2998,7 @@
       ?^  p.sign
         %-  (fail:log %watch-ack 'group watch failed' u.p.sign)
         ?.  (~(has by foreigns) flag)
-          ::TODO  this should not be possible, but if it happens
-          ::      we don't have an invitation, and thus no way to rejoin.
-          ::      the user will still see the group, but it is going
-          ::      to be stale. it would be best to somehow surface
-          ::      it at the frontend.
-          ::
+          %-  (tell:log %crit 'misguided group watch-ack' ~)
           go-core
```

Comment:
This comment remains relevant, right? I'd consider leaving it in for context/detail.